cms/djangoapps/contentstore/views/user.py

```diff
@@ -38,6 +39,7 @@ def request_course_creator(request):
 @login_required
 @ensure_csrf_cookie
 @require_http_methods(("GET", "POST", "PUT", "DELETE"))
+@sudo_required
```

Comment:
Does the Backbone app that calls this view correctly prompt to get sudo access when the user hits this view?